content/subscription/build-details.md

```diff
@@ -33,14 +33,19 @@ If you have an existing [Docker subscription](./details.md), Build Cloud is incl
 - All the features listed for Docker Team plus Build Cloud
 - 800 build minutes every month
 
-## Docker Build Cloud Paid
+## Docker Build Cloud Team
 
 You can upgrade the starter plan for any tier to enhance your developers' experience with the following features:
 
 - 200 additional build minutes per seat
 - Option to buy reserve minutes
 - Increased shared cache
 
+The Docker Build Cloud Team subscription is tied to a Docker organization.
```

Comment:
May be worth linking to the org docs https://docs.docker.com/admin/organization/